templates/operations.html

```diff
@@ -760,15 +760,23 @@ <h1 id="time-tactic" class="member-title"></h1>
         function loadResults(data){
             if (data) {
                 let res = atob(data.output);
-                data.link.facts.sort(function(a, b) { return b.value.length - a.value.length });
-                $.each(data.link.facts, function (k, v) {
-                    res = res.replaceAll(v.value, "<span class='highlight'>" + v.value + "</span>");
+                $('#resultView').text(res);
+                resultText = $('#resultView').html();
+
+                let factValues = Array.from(data.link.facts, fact => fact.value.toString());
+                factValues = Array.from(new Set(factValues));
+                factValues.sort(function(a, b) { return b.length - a.length });
+                $.each(factValues, function (i, factValue) {
```

Comment:
For consistency in this function, can we switch L768 and L769 to use arrow functions (like on 766)? FWIW the styleguide we're experimenting with in the training plugin [prefers arrow functions](https://github.com/airbnb/javascript#arrow-functions).